src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1114,6 +1114,11 @@ public sealed class ValidatePatternAttribute : ValidateEnumeratedArgumentsAttrib
         public RegexOptions Options { set; get; } = RegexOptions.IgnoreCase;
 
         /// <summary>
+        /// Gets or sets the custom error message that is displayed to the user
+        /// </summary>
+        public string ErrorMessage { get; set; }
```

Comment:
The problem is - you can't possible fix every attribute - it's a public abstract class with implementations outside of PowerShell.